app/src/main/java/com/honeycake/tictactoe/ui/composable/GameTitle.kt

```diff
@@ -0,0 +1,17 @@
+package com.honeycake.tictactoe.ui.composable
+
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.text.style.TextAlign
+import com.honeycake.tictactoe.ui.theme.TextColor
+import com.honeycake.tictactoe.ui.theme.Typography
+
+@Composable
+fun GameTitle(){
```

Comment:
composable function just to call composable function without any changes !, i think no need to seperate it ?!

Reply:
I used it for each screen, So it somehow reduces the repeat of Text() composable for each screen I can just call GameTitle()